dev/integration_tests/ios_app_with_extensions/pubspec.yaml

```diff
@@ -19,9 +19,13 @@ dependencies:
   flutter:
     sdk: flutter
 
+  device_info: 0.4.2+7
```

Comment:
Adding (any) plugin is what kicks the tool into running `pod install`, which we need to embed the watch framework.